modules/plugin/coverage-multidim/netcdf/src/test/java/RoundTripper.java

```diff
@@ -0,0 +1,14 @@
+import javax.measure.Unit;
+import org.geotools.imageio.netcdf.NetCDFUnitFormat;
+
+public class RoundTripper {
+
+    public static void main(String[] args) {
+        String unit = "µmol*m^-2*446.2";
```

Comment:
Is that class needed?
Main + system.out :)

Reply:
Oh no, this was just a demo class that I used to report bugs to the UOM library, I'll remove